lib/src/base-styles/_nx-lists.scss

```diff
@@ -142,6 +146,11 @@
       background-color: #d3e6ff;
     }
 
+    &:focus {
+      box-shadow: 0 0 3px 1px rgba(2, 166, 217, 0.3);
```

Comment:
In the nx-table branch I found that this is equivalent to `#02a6d9` with transparency, so I'd rather see it written as `transparentize(#02a6d9, 0.7);`.  That way when we replace the color literals with variables the search-and-replace will pick it up.

Reply:
> In the nx-table branch I found that this is equivalent to `#02a6d9` with transparency, so I'd rather see it written as `transparentize(#02a6d9, 0.7);`. That way when we replace the color literals with variables the search-and-replace will pick it up.

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/e57ab0da139398c2561b16f90a4e9976f7da0c73